src/ext-test/java/org/opentripplanner/ext/fares/impl/FaresIntegrationTest.java

```diff
@@ -45,7 +44,9 @@ public void testBasic() {
     var to = GenericLocation.fromStopId("Destination", feedId, "Mountain View Caltrain");
 
     ItineraryFares fare = getFare(from, to, start, serverContext);
-    assertEquals(fare.getFare(FareType.regular), Money.usDollars(4.25f));
+    var product = fare.getLegProducts().values().iterator().next().product();
+    assertEquals(Money.usDollars(4.25f), product.price());
```

Comment:
Why is this test asserting attribute by attribute whereas the other ones print the entire leg product?

Reply:
I've removed the inconsitency in a8762307d6ad1daae85737de58a67d2039e983c7